### PR TITLE
Signup page

### PR DESCRIPTION
**Pull request by Karabo-Kgatla**
This closes issue #32

### CODE REVIEW

**Comment by Njabuloo** [comment]
The conflict was fixed.

File: views/signup.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+  </div>
+
+  <!--Bootstrap Bundle with Popper -->
+  <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/js/bootstrap.bundle.min.js"
+    integrity="sha384-ka7Sk0Gln4gmtz2MlQnikT1wXgYsOg+OMhuP+IlRH9sENBO0LRn5q+8nbTov4+1p"
+    crossorigin="anonymous"></script>
+
+</body>
+
+</html>
```
Comment (BrianConf) [inline views/signup.html:61]: The sign up page is fine, so I will merge it into the main branch

**Review by BrianConf — APPROVED** [review]
The sign up page is fine, thus I will merge it into the main branch

**Comment by BrianConf** [comment]
Noted resolved conflicts

**Pull request merged**